### ISSUE TITLE
[Desktop Mode] A black screen is displayed when playing 360 videos from Youtube in immersive mode

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2816]
**[Affected Versions]:**
- Firefox Reality 9 RC 2 (37ac4cbd)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- Oculus Go - Android 7.1.2
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2
- Pico Neo 2 - Android - 8.1.0

**[Prerequisites]:**
- The headset is turned on, the browser is installed and open.

**[Steps to reproduce]:**										
1. Change to Desktop Mode from the Hamburger Menu.
2. Navigate to Youtube website. 
3. Play a 360 video in immersive mode.
4. Observe the behavior.

**[Expected results]:**
- The video is played correctly.

**[Actual results]:**
- A black screen is displayed.

**[Notes]:**
- I was not able to reproduce this video with videos from Vimeo website.
- The Sound of the video is not affected.
- This issue is also reproducible in the production version (v8.1) of the app.
- Attached is screen recording of the issue: [link](https://drive.google.com/open?id=1Fh6YcjWm8GHM5ioNkbTqRndCsQhO_ArP).

**Comment by heres-maria** [comment]
The verification of this issue is currently blocked by #2885.

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the Firefox Reality 9 rc6 (abe931f) build. The black screen is no longer displayed when playing Youtube videos in immersive mode while using Desktop mode from the Hamburger menu.

Tested on Oculus Go (Android 7.1.2), Oculus Quest (Android 7.1.1), HTC Vive Focus Plus (Android 7.1.2), HTC Vive Focus (Android 7.1.2) and Pico Neo 2 (Android 8.1.0).
